tests/integration/mutation/create/embeddings/embedding_test.go

```diff
@@ -0,0 +1,163 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+//go:build linux
```

Comment:
todo: Please document in this file why these tests are Linux only.  I know you noted this in the PR description, but if this constraint persists it will take some figuring out as to why it is Linux only.

suggestion: Consider creating a ticket and linking it to and from here to remove this constraint.

Reply:
Changed the way they are executed.